src/views/EnrollmentCheckIn/components/RaceEthnicity/index.jsx

```diff
@@ -11,10 +10,9 @@ import {
 } from '@mui/material';
 
 const RaceEthnicity = ({ demographic, handleChangeDemographic, handleCheckDemographic }) => {
-  const cyan = gordonColors.primary.cyan;
   return (
     <Grid container justifyContent="center" alignItems="center" direction="column" spacing={1}>
-      <Grid item style={{ color: cyan }}>
+      <Grid item color="secondary">
```

Comment:
Similarly, `Grid` has no 'color' prop.

Reply:
I moved the color prop down into the typography, which I think is what was intended originally.  (See above comment for usage of color in Typography)